engine/src/main/java/org/terasology/rendering/dag/nodes/InitialPostProcessingNode.java

```diff
@@ -83,17 +95,24 @@ public InitialPostProcessingNode(Context context) {
 
         initialPostMaterial = getMaterial(INITIAL_POST_MATERIAL);
 
+        renderingConfig = context.get(Config.class).getRendering();
+        isBloom = renderingConfig.isBloom();
+        renderingConfig.subscribe(RenderingConfig.BLOOM, this);
+        isLightShafts = renderingConfig.isLightShafts();
```

Comment:
Let's rename these variables to:
- bloomIsEnabled
- lightShaftsAreEnabled

please.